vocabularies/Analytics.xml

```diff
@@ -104,7 +141,29 @@
         </Annotation>
       </ComplexType>
 
+      <Term Name="AnalyticalContext" Type="Collection(Analytics.AnalyticalContextType)" Nullable="false" AppliesTo="EntitySet Collection">
+        <Annotation Term="Core.AppliesViaContainer"/>
+        <Annotation Term="Core.Description" String="Collection of properties that define an analytical context" />
+      </Term>
+      <ComplexType Name="AnalyticalContextType">
+        <Property Name="Property" Type="Edm.PropertyPath" Nullable="false">
+          <Annotation Term="Core.Description" String="Property that is part of the analytical context" />
+        </Property>
+        <Property Name="Dimension" Type="Core.Tag" Nullable="false" DefaultValue="true">
+          <Annotation Term="Core.Description" String="The property holds the key of a dimension" />
+        </Property>
+        <Property Name="Measure" Type="Core.Tag" Nullable="false" DefaultValue="true">
+          <Annotation Term="Core.Description" String="The property holds the numeric value of a measure" />
+        </Property>
+        <Property Name="AccumulativeMeasure" Type="Core.Tag" Nullable="false" DefaultValue="true">
+          <Annotation Term="Core.Description" String="The measure has non-negative and additive values; it can be used in whole-part charts, e.g. the Donut" />
+        </Property>
+      </ComplexType>
+
```

Comment:
```suggestion
```

Reply:
Current proposal is cleaner, because the deprecated and the future terms are separated.